adapters/repos/db/aggregator/grouper.go

```diff
@@ -135,6 +135,10 @@ func (g *grouper) addElementById(s *models.PropertySchema, docID uint64) error {
 		for i := range val {
 			g.addItem(val[i], docID)
 		}
+	case []interface{}:
+		for i := range val {
+			g.addItem(val[i], docID)
```

Comment:
it says this line is not covered by tests? (I could swear it didn't show that 5min ago)

Reply:
:)